src/o0baseauth.h

```diff
@@ -117,6 +117,11 @@ public Q_SLOTS:
     /// Set extra tokens found in OAuth response
     void setExtraTokens(QVariantMap extraTokens);
 
+    /// Set local reply server
+    void setReplyServer(O2ReplyServer *server);
+
+    O2ReplyServer * replyServer() const;
+
 protected:
     QString clientId_;
```

Comment:
I think protected members are bad idea (#114 proves that). So it will be great to replace them with protected getter / setter